src/cargo/util/toml/mod.rs

```diff
@@ -60,6 +61,12 @@ pub fn read_manifest(
     let mut warnings = Default::default();
     let mut errors = Default::default();
 
+    debug!(
+        "read_manifest; path={}; source-id={}",
+        path.display(),
+        source_id
+    );
```

Comment:
These already get logged at the `trace` level

Reply:
I plan on cleaning out debug logging in the final PR.

Reply:
This one is removed. I'm leaving some debug and trace logging that I think makes sense.